src/pylhe/__init__.py

```diff
@@ -288,9 +288,10 @@ def read_lhe(filepath):
         with _extract_fileobj(filepath) as fileobj:
             for event, element in ET.iterparse(fileobj, events=["end"]):
                 if element.tag == "event":
-                    data = element.text.split("\n")[1:-1]
+                    data = element.text.strip().split("\n")
```

Comment:
@eduardo-rodrigues to answer your question on if this is good to merge, these two sections are uncovered still. It is late for me at the moment, but I can look at adding tests tomorrow unless you beat me to it. (I'm catching up on a backlog of work from holiday and being sick).

Reply:
Sorry to hear that. Hope nothing too serious.

Seen you've changed job. Congrats!

Reply:
[Today is going to be difficult for me to do developments.]

Reply:
How do you want to test this trivial line `data = element.text.strip().split("\n")`, @matthewfeickert ?

Reply:
Will try and look into this to finalise asap ... For the moment I've made this branch in sync with master, as it had diverged quite a bit.

Reply:
Done.